manticore/ethereum.py

```diff
@@ -1175,14 +1172,14 @@ def flagged(flag):
                         summary.write("\t%032x -> %032x %s\n" % (offset, value, flagged(is_storage_symbolic)))
                         is_something_symbolic = is_something_symbolic or is_storage_symbolic
 
-                code = blockchain.get_code(account_address)
-                if len(code):
+                runtime_code = blockchain.get_code(account_address)
+                if runtime_code:
                     summary.write("Code:\n")
-                    fcode = StringIO.StringIO(code)
+                    fcode = StringIO.StringIO(runtime_code)
```

Comment:
Approving this PR, but we should make sure our uses of `str` vs `bytes` (and `StringIO.StringIO` vs `io.BytesIO`) are consistent as we continue our port to py3. (cc @f2xeb)

Reply:
yeah in general BytesIO is very likely what you want unless you know you're dealing with a unicode encoded string